trace/execution_icons.go

```diff
@@ -0,0 +1,102 @@
+package trace
+
+import (
+	"github.com/fatih/color"
+	"go.temporal.io/api/enums/v1"
+)
+
+var (
```

Comment:
All of this is great, I would also add spinners on anything that's currently in progress.
The colors are awesome but not super intuitive to me and don't account for color blindness.

Reply:
I'm not sure to understand what "in progress" means in this context.

I completely understand the concern with colour blindness, but there's so much I've been able to address for now. I'm happy to learn about how to improve the tool in this regard.

Reply:
In progress would be any activity/child workflow/timer that has not yet been concluded (e.g. completed/failed/cancelled...)
I'll open an issue to reconsider the design.